src/System.Net.Http/tests/FunctionalTests/ManagedHandlerTest.cs

```diff
@@ -266,6 +266,108 @@ public override int Read(byte[] buffer, int offset, int count)
         }
     }
 
+    public sealed class ManagedHandler_Authentication_Test : HttpClientTestBase
+    {
```

Comment:
Wouldn't we want these tests to be in HttpClientHandlerTest etc. so that they can be exercised by all the handlers?

Reply:
I ran these with winhttp handler, and there are a lot of test failures, for one, winhttp handler doesn't support Digest, and there were some failures due to non-conformance with RFC.

Reply:
>winhttp handler doesn't support Digest

In general WinHttpHandler supports all the auth schemes that WinHTTP supports including Basic, Digest, Negotiate, NTLM.  So, if there is a bug with Digest for WinHttpHandler, we should open an issue, etc.

Reply:
> I ran these with winhttp handler

What about CurlHandler? If they both don't work, maybe the issue is actually with the server implementation rather than the client?